guide/interactions/autocomplete.md

```diff
@@ -0,0 +1,110 @@
+# Autocomplete
+
+Slash command options provide the feature of autocomplete. In this section, we'll cover how to add autocomplete support to your command and how to respond to it.
+
+::: tip
+This page is a follow-up to the [interactions (slash commands) pages](/interactions/registering-slash-commands.md). Please carefully read those first so that you can understand the methods used in this section.
+:::
+
+::: warning
+Make sure that you're on discord.js version 13.3.0 or above before using autocomplete.
+:::
+
+## Preparing commands
+
+To use autocomplete with your command options, you have to tell Discord to enable it.
+How to do this is dependent on how you create your slash commands.
+
+### Slash command builder
+
+The slash command builder does not yet support using autocomplete.
+
+### Custom system
+
+For other custom systems or entering it raw, your resulting code should look something like this. To enable autocomplete, set it to true in the option.
```

Comment:
```suggestion
```
Calling this a "custom system" feels out of place, what defines custom? This just an object with a property that holds a value, this can just be straightforward used as example.

Reply:
Should this be removed with the merge of the slash command builder which makes it now support autocomplete?

Reply:
If a stable builders release support it, use builders for consistency, imo